src/flask_pyoidc/provider_configuration.py

```diff
@@ -128,6 +129,8 @@ class ProviderConfiguration:
     """
 
     DEFAULT_REQUEST_TIMEOUT = 5
+    DEFAULT_CACHE_MAXSIZE = 1024
```

Comment:
To make it easily configurable these should be added as parameters to the constructor, something like (pseudo-code, needs to fix actual types):
```python
ProviderConfiguration(
  ...,
  introspection_cache_configuration: {max_size: int, max_ttl: int}
```